src/pages/index.js

```diff
@@ -0,0 +1,314 @@
+const profileDescription = document.querySelector(".profile__description");
+const avatarModalButton = document.querySelector(".profile__avatar-btn");
+
+// Modal Elements
+const editProfileModal = document.querySelector("#edit-profile-modal");
+const editFormElement = editProfileModal.querySelector(".modal__form");
+const editModalCloseButton =
+  editProfileModal.querySelector(".modal__close-btn");
+const editModalNameInput = editProfileModal.querySelector(
+  "#profile-name-input"
+);
+const editModalDescriptionInput = editProfileModal.querySelector(
+  "#profile-description-input"
+);
+
+// Card Elements
+const cardModal = document.querySelector("#add-card-modal");
+const cardForm = cardModal.querySelector(".modal__form");
+const cardModalCloseBtn = cardModal.querySelector(".modal__close-btn");
+const cardSubmitButton = cardModal.querySelector(".modal__submit-btn");
+const cardNameInput = cardModal.querySelector("#add-card-name-input");
+const cardLinkInput = cardModal.querySelector("#add-card-link-input");
+
+let selectedCard, selectedCardId;
+
+// Avatar Elements
+const avatarModal = document.querySelector("#edit-avatar-modal");
+const avatarForm = avatarModal.querySelector(".modal__form");
+const avatarLinkInput = avatarModal.querySelector("#profile-avatar-input");
+const avatarModalCloseBtn = avatarModal.querySelector(".modal__close-btn");
+const avatarSubmitButton = avatarModal.querySelector(".modal__submit-btn");
+
+// Preview Modal elements
+const previewModal = document.querySelector("#preview-modal");
+const previewModalImageEl = previewModal.querySelector(".modal__image");
+const previewModalCaptionEl = previewModal.querySelector(".modal__caption");
+const previewModalCloseBtn = previewModal.querySelector(
+  ".modal__close-btn_type_preview"
+);
+
+// Delete Form Elements
+const deleteModal = document.querySelector("#delete-modal");
+const deleteModalCloseButton = deleteModal.querySelector(".modal__close-btn");
+const deleteForm = deleteModal.querySelector(".modal__form");
+const deleteFormCancelBtn = deleteModal.querySelector(".modal__cancel-btn");
+
+// Load Images
+document.getElementById("pencil").src = pencilSrc;
+document.getElementById("logo").src = logoSrc;
+document.getElementById("plus").src = plusSrc;
+document.getElementById("bessie-coleman").src = colemanSrc;
+
+// Handle Add Card
+function handleAddCardSubmit(evt) {
+  evt.preventDefault();
+
+  const inputValues = {
+    name: cardNameInput.value,
+    link: cardLinkInput.value,
+  };
+
+  setButtonText(evt.submitter, "Saving...");
+
+  api
+    .addCard(inputValues)
+    .then((cardData) => {
+      const cardElement = getCardElement(cardData);
+      cardList.prepend(cardElement);
+      closeModal(cardModal);
+      evt.target.reset();
+      disableButton(cardSubmitButton, settings);
+    })
+    .catch(console.error)
+    .finally(() => {
+      setButtonText(evt.submitter, false, "Saving...", "Save");
```

Comment:
`"Saving...", "Save"` are not needed here. There is a mistake in the order. It should be `"Save", "Saving..."`


But the 3rd and 4th arguments are stated by **default** inside `setButtonText`, so you don't need to repeat it here


```js
    .finally(() => {
      setButtonText(evt.submitter, false);
    });

```